### PR TITLE
add clarification on configuring postgres datadog database monitoring

### PR DESCRIPTION
**Pull request by runwuf**
add clarification on configuring postgres datadog database monitoring as customer was confused if they also need to follow the instructions from datadog

### CODE REVIEW

**Comment by lornajane** [comment]
Could we say that these instructions replace those ones, or that we've got the user to the same place as those instructions? I think if we can add some explanation of why it isn't needed, that would help.

**Comment by lornajane** [comment]
@runwuf could you take a quick look at my question on this? I don't think we can write instructions and then tell the user not to follow them without explaining more! And @divdby0 it would be good to get your eyes on this as well, please

**Comment by runwuf** [comment]
> @runwuf could you take a quick look at my question on this? I don't think we can write instructions and then tell the user not to follow them without explaining more! And @divdby0 it would be good to get your eyes on this as well, please

Updated.  Thanks

**Comment by staceysalamon-aiven** [comment]
@wojcik-dorota Would you review please?

File: docs/products/postgresql/howto/monitor-database-with-datadog.rst
Comment (wojcik-dorota) [inline docs/products/postgresql/howto/monitor-database-with-datadog.rst:49]: > After completed the above steps successfully, Datadog Database Monitoring setup is ready.

``Executing the steps successfully results in enabling Datadog Database Monitoring for your service.``

> These instructions replaced the steps in `Datadog Database Monitoring setup for PostgreSQL® <https://docs.datadoghq.com/database_monitoring/setup_postgres/>`_.

How about including this info at the beginning of the article (after the info paragraphs and before Prerequisites, for example?

Also, what this instruction is actually replacing? What steps in particular you mean? Have there been any Aiven-specific instructions at https://docs.datadoghq.com/database_monitoring/setup_postgres/?
If so, maybe rephrase into ``This article replaces the instruction on how to set up Datadog Database Monitoring for Aiven for PostgreSQL that used to be available in `Setting up PostgreSQL <https://docs.datadoghq.com/database_monitoring/setup_postgres/>`_.``

**Review by wojcik-dorota — CHANGES_REQUESTED** [review]

File: docs/products/postgresql/howto/monitor-database-with-datadog.rst
Comment (runwuf) [reply docs/products/postgresql/howto/monitor-database-with-datadog.rst]: updated as per your suggestion, thanks!

**Comment by wojcik-dorota** [comment]
@divdby0 could you please approve if ok?

**Review by wojcik-dorota — APPROVED** [review]

**Comment by wojcik-dorota** [comment]
@runwuf , I approved it only from my pov and requested another review (see the comments) before merging that from @divdby0 . My bad - I guess I shouldnt have approved until all the other required reviewers approved.

As confirmed with @divdby0 , this PR required fixing:

```
The article doesn’t replace the instruction in “[Setting up PostgreSQL](https://docs.datadoghq.com/database_monitoring/setup_postgres/).”
Check the bullet 2nd. The “Ensure the [Datadog Metrics integration](https://3ea5eace.devportal.pages.dev/docs/integrations/datadog/datadog-metrics.html) is enabled.”
It is a prerequisite actually.
How about pulling the 2nd bullet from the prerequisites and replace it on top of this “Replace” sentence?
This is not replacing it but this is in fact needing it.
```

**Comment by divdby0** [comment]
Database monitoring with Datadog is a standalone feature on Aiven which extends over the standard DataDog integration.
The standard DataDog integration is a prerequisite of this feature which was highlighted in the 2nd bullet of the Prerequisites section of this article. A "replaces" indication is misleading, IMHO.
I recommend addressing the customer concern differently.

**Pull request merged**
